jenkins.go

```diff
@@ -205,12 +214,44 @@ func (jenkins *Jenkins) CreateView(listView ListView) error {
 
 // Create a new build for this job.
 // Params can be nil.
-func (jenkins *Jenkins) Build(job Job, params url.Values) error {
+//
+// If the build was successfully started, the first response value will be the
+// Queue ID of the new build - or -1 if it could not be read.
+func (jenkins *Jenkins) Build(job Job, params url.Values) (int, error) {
+	var path string
 	if hasParams(job) {
-		return jenkins.post(fmt.Sprintf("/job/%s/buildWithParameters", job.Name), params, nil)
+		path = fmt.Sprintf("/job/%s/buildWithParameters", job.Name)
 	} else {
-		return jenkins.post(fmt.Sprintf("/job/%s/build", job.Name), params, nil)
+		path = fmt.Sprintf("/job/%s/build", job.Name)
+	}
+
+	resp, err := jenkins.postForResponse(path, params, nil)
+	if err != nil {
+		return -1, err
```

Comment:
if the second return value error is not nil, the first should not be used.  So the initial value 0 of int is enough to use if the second is not nil.